Fabric.Identity.API/scripts/Install-IdPSS-Utilities.psm1

```diff
@@ -300,52 +300,55 @@ function Set-IdentityProviderSearchServiceWebConfigSettings {
         [System.Security.Cryptography.X509Certificates.X509Certificate2] $encryptionCert,
         [string] $appName
     )
-
-    # Alter IdPSS web.config for azure
-    $clientSettings = @()
-    $clientSettings += Get-ClientSettingsFromInstallConfig -installConfigPath $installConfigPath -appName $appName
-
-    Clear-IdentityProviderSearchServiceWebConfigAzureSettings -webConfigPath $webConfigPath
+	Clear-IdentityProviderSearchServiceWebConfigAzureSettings -webConfigPath $webConfigPath
```

Comment:
@hckenmiller , @ctediz - what is the purpose of clearing the settings first? Asking because during a deployment, the web.config is overwritten in the upgrade scenario, so we are always starting from a known state.

Reply:
Running install-identity, shouldn't the azuresettings getting cleared out if they changed from $useAzure = $true to $false in the install.config?

Reply:
I guess what I am trying to communicate is that every time we run an installation, we blow away the entire web.config and it gets re deployed with our default settings, so we start from a known state, which doesn't include any Azure settings. So I am not sure there is anything to clear...

Reply:
Ok. I get it. I was just basing it upon running install-identity. If that is the case then I could probably remove it.

Reply:
I think I know what happened, it clears out for identity but not idpss since this script doesn't install idpss, so the web.config wasn't in a clean state when updating the idpss web.config

Reply:
Found issue running install-identity locally, not updating web.config, because of a bad Fabric.Identity.API.zip file. Jason's IdPSS only gets cleared out if you run the DosInstaller IdPSS step, so will leave the clear function for his.